brave/instrumentation-aws-java-sdk-core/src/test/java/brave/instrumentation/aws/AwsClientTracingTest.java

```diff
@@ -25,27 +25,28 @@
 import com.amazonaws.services.dynamodbv2.model.AttributeValue;
 import com.amazonaws.services.s3.AmazonS3;
 import com.amazonaws.services.s3.AmazonS3ClientBuilder;
+import java.io.IOException;
 import java.util.Collections;
 import okhttp3.mockwebserver.MockResponse;
 import okhttp3.mockwebserver.MockWebServer;
-import org.junit.Before;
 import org.junit.Rule;
-import org.junit.Test;
 import org.junit.contrib.java.lang.system.EnvironmentVariables;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-public class AwsClientTracingTest extends ITRemote {
-
-  @Rule
+class AwsClientTracingTest extends ITRemote {
   public MockWebServer mockServer = new MockWebServer();
+
+  // TODO: replace this with jupiter
   @Rule
   public EnvironmentVariables environmentVariables = new EnvironmentVariables();
```

Comment:
We've used https://github.com/webcompere/system-stubs for setting environment variables in tests with Jupiter.